2018-02-07 11:43:35,269 [ERROR   ] at my_logging.py <module>.49 - - division by zero
Traceback (most recent call last):
  File "/Users/afa/myFiles/codeDoc/PythonDoc/my_py_notes_万物皆对象/日志处理(log)/logging/my_logging.py", line 46, in <module>
    1/0
ZeroDivisionError: division by zero
